### ISSUE TITLE
Meta-Sim2: Unsupervised Learning of Scene Structure for Synthetic Data Generation

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#62]
# TL;DR
They proposed Meta-Sim2, which generates unsupervised datasets similar to real data. In this method, the scene is structured as a sequence using RNN and optimized by reinforcement learning, which was not possible in our previous work, Meta-Smi. The key is to pre-train with a simple prior (such as "one car").
![image](https://user-images.githubusercontent.com/24524018/91838005-ca7ab180-ec87-11ea-89a7-9adb24d42545.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2008.09092

# Submission Dates(yyyy/mm/dd)
2020/08/20

# Authors and institutions
Jeevan Devaranjan, Amlan Kar, Sanja Fidler
- NVIDIA 
- University of Toronto 
- University of Waterloo 
- Vector Institute

# Methods

# Results

# Comments
